src/panels/calendar/ha-panel-calendar.ts

```diff
@@ -187,7 +187,7 @@ class PanelCalendar extends LitElement {
           .events=${this._events}
           .calendars=${this._calendars}
           .narrow=${this.narrow}
-          .initialView=${this.narrow ? "listWeek" : "dayGridMonth"}
+          initialView="dayGridMonth"
```

Comment:
I don't think `dayGridMonth` works well on small screens, that's why I didn't make it the default there